src/main/scala/io/iohk/ethereum/network/PeerManagerActor.scala

```diff
@@ -154,13 +154,16 @@ class PeerManagerActor(
     NetworkMetrics.DiscoveredPeersSize.set(nodes.size)
     NetworkMetrics.BlacklistedPeersSize.set(blacklistedPeers.size)
     NetworkMetrics.PendingPeersSize.set(connectedPeers.pendingPeersCount)
+    NetworkMetrics.TriedPeersSize.set(triedNodes.size)
 
     log.info(
-      s"Discovered ${nodes.size} nodes, " +
+      s"Discovered ${nodes.size} (can connect to ${discoveredNodes.size}) nodes, " +
         s"Blacklisted ${blacklistedPeers.size} nodes, " +
         s"handshaked to ${connectedPeers.handshakedPeersCount}/${peerConfiguration.maxOutgoingPeers + peerConfiguration.maxIncomingPeers}, " +
         s"pending connection attempts ${connectedPeers.pendingPeersCount}. " +
-        s"Trying to connect to ${nodesToConnect.size} more nodes."
+        s"Trying to connect to ${nodesToConnect.size} more nodes, " +
+        s"Total tried nodes ${triedNodes.size}, " +
+        s"Connected nodes ${connectedPeers.peers.size}."
```

Comment:
This log message becomes a bit confusing like this, for example `Discovered 1154 (can connect to 790) nodes, Blacklisted 426 nodes, handshaked to 6/80, pending connection attempts 4. Trying to connect to 40 more nodes, Total tried nodes 435, Connected nodes 10.`

Reply:
What about something like this? 
`Total number of discovered nodes 1154.
Total number of connection attempts 435, blacklisted 426 nodes, connected to 10 nodes.
Handshaked 6/80, pending connection attempts with 4 nodes. Trying to connect to 40 more nodes.`

Reply:
agree, especially handshaked and connected showing different numbers is confusing.
and generally this looks like too much information

Reply:
Looking in more detail, the `Connected nodes 10` is also confusing it is actually 6 handshaked and 4 pending, so maybe I would remove the  `Connected nodes 10` actually